Pod/Classes/ObjC/URBNValidatorCompat.swift

```diff
@@ -32,7 +32,7 @@ extension CompatValidateable {
 }
 
 @objc public class URBNCompatValidator: NSObject, CompatValidator {
-    private let backingValidator: URBNValidator = URBNValidator()
+    private var backingValidator: URBNValidator = URBNValidator(bundle: NSBundle(forClass: URBNCompatValidator.self))
```

Comment:
Because URBNValidator is a struct now.  And since we're setting the localization bundle it has to be var